integration_tests/CMakeLists.txt

```diff
@@ -551,6 +575,7 @@ RUN(NAME structs_23          LABELS cpython llvm c NOFAST)
 RUN(NAME structs_24          LABELS cpython llvm c)
 RUN(NAME structs_25          LABELS cpython llvm c)
 RUN(NAME structs_26          LABELS cpython llvm c)
+RUN(NAME symbolics_01        LABELS cpython sympy)
```

Comment:
I think this might currently fail at the CI (in `LPython CI / LPython CI (3.10, ubuntu-latest) (pull_request)`). At the `CI`, I think we do not have `sympy` installed. You need to run this test only in the Job `LPython CI / Run SymPy tests (pull_request)`, where you have `sympy` installed.

Reply:
I think this can be supported by following https://github.com/lcompilers/lpython/pull/1846#discussion_r1232530277.

Reply:
Correct I guess this commit (https://github.com/lcompilers/lpython/commit/bd7e0d4593ddfac2aacecc8e03f46fe98558d831) did install sympy in the environment but it was taken out during the rebase !

Reply:
And as suggested, sympy might not be a good addition to the environment due to reasons stated here https://github.com/lcompilers/lpython/pull/1846#discussion_r1232528749

Reply:
Checkout these comments too 
https://github.com/lcompilers/lpython/pull/1846#discussion_r1232527167
https://github.com/lcompilers/lpython/pull/1846#discussion_r1232530277

This is the direction we would like to go for but I am not fully sure on how to proceed .

Reply:
Please checkout the commit [CI: Add separate job for CPython Interop tests](https://github.com/lcompilers/lpython/pull/1933/commits/56db034c8b54ba5d47d62c51dd2ec4ad6d193b58). I think it could guide you.

Reply:
Sure ! Thanks , I'll go through it and get back !